src/main/java/florizz/objects/Flower.java

```diff
@@ -165,9 +165,9 @@ public String getFlowerName() {
      * @return The name and colour of the flower.
      */
     public String getNameAndColour() {
-        return colour.toString().charAt(0) +
-                colour.toString().replaceAll("_", " " ).substring(1).toLowerCase() + " " +
-                name;
+        return name + " (" + colour.toString().charAt(0) +
+                colour.toString().replaceAll("_", " " ).substring(1).toLowerCase() +
+                ")";
```

Comment:
Should make it clearer they dont have to input the colour  of the flower anymore good job!